nextflow.config

```diff
@@ -6,34 +6,30 @@
 ----------------------------------------------------------------------------------------
 */
 
-// Wave containers
-// Docker: Comment in for ARM64 build:
-// process.container = 'community.wave.seqera.io/library/pip_drevalpy:23d68e736486ab16'
-// Docker: Comment in for AMD64 build:
-// process.container = 'community.wave.seqera.io/library/pip_drevalpy:7d8f30cda5f4887b'
-// Singularity: Comment in for AMD64 build:
-// process.container = 'oras://community.wave.seqera.io/library/pip_drevalpy:6ca244fae0c2fa29'
+process.container = 'ghcr.io/daisybio/drevalpy:main'
+process.conda = 'env.yml'
 
 // Global default params, used in configs
 params {
     // For this pipeline
     run_id = 'my_run'
```

Comment:
Yes, I just need one container or conda environment for all processes because they all depend on the same package. Is there a better place to set this?

Reply:
The container and conda settings are fine. 

Matthias is referring to the `params.run_id`. For example, you could use `workflow.sessionId` or `workflow.runName` here instead. I think the question is more, "Why is run_id a pipeline parameter?".

Reply:
ahh, yes, thanks for clarifying this. yes, that's exactly what I meant.

Reply:
Ah, I see @mahesh-panchal! The idea was that people can, e.g., compare different models with each other in different runs, such that they get separate results for them. They might, e.g., just want to compare baseline models against one state-of-the-art model in one run but then, in another run, they want to do an ablation study of one state-of-the-art model and just compare those results against a naive baseline. Then those results would get saved into different subdirectories. Is there a more nextflow way to do this?

Reply:
I think this is generally fine then. It's better to have a user-defined informative name than an uninformative string of numbers/letters and/or unrelated names. 

It's possible to do this all in one such that you supply inputs with an id, parameters, etc, and then use meta map to propagate this information so you can run it all at once, but that can be left as a future pipeline enhancement.

Reply:
I'll add it as an issue!